lib/mi_bridges/driver/housing_utility_bills_page.rb

```diff
@@ -26,20 +26,28 @@ def continue
       private
 
       def check_housing_bills
-        check_in_section "starHousingBills", if: true, for: first_name
+        check_in_section(
+          "starHousingBills",
+          condition: true,
+          for_name: first_name,
+        )
       end
 
       def check_utility_bills
-        check_in_section "starUtilityBills", if: true, for: first_name
+        check_in_section(
+          "starUtilityBills",
+          condition: true,
+          for_name: first_name,
+        )
       end
 
       def check_room_and_meals
         check_in_section "starRoomandMeals"
       end
 
       def select_home_heating_payment
-        check_in_section "heatingResponseGroup", if: true, for: "No"
-        check_in_section "meapReceivedGroup", if: true, for: "No"
+        check_in_section "heatingResponseGroup", condition: true, for_name: "No"
+        check_in_section "meapReceivedGroup", condition: true, for_name: "No"
```

Comment:
*Most* of the time the check box is for a name, but sometimes it's not. Sometime's it's something like "No" or "None".  Is there something else that might be better? Maybe `for_label`?

Reply:
Great point! Renamed at https://github.com/codeforamerica/michigan-benefits/commit/d504e3716a1dc1dc09a5a669ecd99b753ec6db05